### ISSUE TITLE
remove_flow in BPMN model not removing edge from graph

### ISSUE THREAD

**Issue by YJDave** [process-intelligence-solutions/pm4py#192]
Hi,

I am trying to remove the flow from BPMN model using `bpmn.remove_flow()` function. It successfully removes the flow from BPMN model, however when I plot the BPMN model graph using `bpmn.get_graph()` function it still shows the edge/flow I removed. 

For example below, 
```python3

## Initialization
bpmn = BPMN()
start = BPMN.Task(name="start")
a = BPMN.Task(name="a")
b = BPMN.Task(name="b")
f1 = BPMN.Flow(start, a)
f2 = BPMN.Flow(a, b)
f3 = BPMN.Flow(b, start)
bpmn.add_flow(f1)
bpmn.add_flow(f2)
bpmn.add_flow(f3)

# Flows: Previous
print(bpmn.get_flows())
"""
{id0fda534e-0342-459e-a2c6-93414c333af7@start -> id54e0e565-e06c-45d7-95b3-61035fa2b2ed@a,
 id1d23ba4c-a10e-4c01-a16b-6cc0981f5173@b -> id0fda534e-0342-459e-a2c6-93414c333af7@start,
 id54e0e565-e06c-45d7-95b3-61035fa2b2ed@a -> id1d23ba4c-a10e-4c01-a16b-6cc0981f5173@b}
"""

# Remove the flow
bpmn.remove_flow(f1)

# Flow is removed from bpmn model as we can see in the list
print(bpmn.get_flows())
"""
{id1d23ba4c-a10e-4c01-a16b-6cc0981f5173@b -> id0fda534e-0342-459e-a2c6-93414c333af7@start,
 id54e0e565-e06c-45d7-95b3-61035fa2b2ed@a -> id1d23ba4c-a10e-4c01-a16b-6cc0981f5173@b}
"""

# However the edge is still there in graph
print(bpmn.get_graph().edges)
"""
OutEdgeView([(0fda534e-0342-459e-a2c6-93414c333af7@start, 54e0e565-e06c-45d7-95b3-61035fa2b2ed@a), 
(54e0e565-e06c-45d7-95b3-61035fa2b2ed@a, 1d23ba4c-a10e-4c01-a16b-6cc0981f5173@b), 
(1d23ba4c-a10e-4c01-a16b-6cc0981f5173@b, 0fda534e-0342-459e-a2c6-93414c333af7@start)])
"""
```

The reason I am getting the graph from BPMN model is because I want to plot the model/graph. I am also getting the edge in graph I plotted which is very obvious, since the edge is present in graph object. 
Is this expected behaviour? If yes, then is there a way I can make this consistent for my use case? If not, am I doing something wrong here?

**Comment by fit-alessandro-berti** [comment]
Dear YJDave,

Thanks for your issue. Actually in remove_flow, the flow was removed from the BPMN object itself but not from the underlying graph object.

We will release a fix for this in the next release.

**Comment by YJDave** [comment]
Thank you, on this note, `remove_node` seems to work as expected, it removes node from graph object.

**Comment by YJDave** [comment]
Submitted quick fix PR #193
